apps/vmq_server/src/vmq_queue.erl

```diff
@@ -195,11 +192,17 @@ save_sync_send_event(Queue, Event) ->
             Ret
     end.
 
-status(Queue) ->
-    gen_fsm:sync_send_all_state_event(Queue, status, infinity).
-
-info(Queue) ->
-    gen_fsm:sync_send_all_state_event(Queue, info, infinity).
+save_sync_send_all_state_event(Queue, Event) ->
+    case catch gen_fsm:sync_send_all_state_event(Queue, Event, infinity) of
+        {'EXIT', {normal, _}} ->
+            {error, noproc};
+        {'EXIT', {noproc, _}} ->
+            {error, noproc};
+        {'EXIT', Reason} ->
+            exit(Reason);
```

Comment:
What's the reason for removing these clauses? Why aren't they needed? Same comment for the ones in get_opts?

Reply:
aah, these were moved... the diff between + and - and red and green...